examples/raw-hello-world/src/main/resources/application.conf

```diff
@@ -0,0 +1,30 @@
+# This is still the default configuration but it's good to see what can be changed
+mesos-client {
+
+  master-url: "127.0.0.1:5050"
```

Comment:
What do you think about using an environment variable here?

Reply:
Why? It's an example, nobody will use it in production, right? 🤞